cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -162,6 +164,11 @@ private boolean isKafkaAssignerMode(List<String> goals) {
     ModelCompletenessRequirements modelCompletenessRequirements =
         modelCompletenessRequirements(goalsByPriority.values()).weaker(requirements);
     try (AutoCloseable ignored = _loadMonitor.acquireForModelGeneration(operationProgress)) {
+      Cluster cluster = _loadMonitor.refreshClusterAndGeneration().cluster();
```

Comment:
Can we move this check to a util function -- e.g. `sanityCheckBrokerPresence(Set<Integer> brokers)`?